$NetBSD$

Use Ruby/GStreamer in Ruby/Gnome2 intead of deprecated ruby gstreamer0.10.

--- clutter-gst/rbcluttergst.c.orig	2008-08-05 14:32:00.000000000 +0000
+++ clutter-gst/rbcluttergst.c
@@ -52,7 +52,7 @@ Init_clutter_gst ()
 {
   rb_require ("glib2");
   rb_require ("clutter");
-  rb_require ("gst0.10");
+  rb_require ("gst");
 
   /* The base class init of some of the clutter-gst classes call some
      gst functions so unfortunatly defining the Ruby wrappers will
